docs/en/latest/config.json

```diff
@@ -212,6 +213,14 @@
             "discovery/kubernetes"
           ]
         },
+        {
+          "type": "category",
+          "label": "Pub-Sub",
```

Comment:
Better always use the same format "PubSub"

Reply:
changed